cmd/skopeo/inspect.go

```diff
@@ -185,3 +187,15 @@ func (opts *inspectOptions) run(args []string, stdout io.Writer) (retErr error)
 	fmt.Fprintf(stdout, "%s\n", string(out))
 	return nil
 }
+
+func retryGetManifest(ctx context.Context, instanceDigest *digest.Digest, maxRetry int, retryDelay time.Duration, src types.ImageSource) ([]byte, error) {
```

Comment:
This would very soon pay off writing as a generic utility,
```go
func retryIfNecessary(ctx context.Context, operation func() error, retryOptions) error
```
or something like that.